clients/src/main/java/org/apache/kafka/clients/NetworkClient.java

```diff
@@ -109,6 +111,16 @@
 
     private final Sensor throttleTimeSensor;
 
+    /**
+     *  KIP-219: When communicating with the nodes contained in this set, if a response with a non-zero throttle time is
```

Comment:
There are a lot of comments like this that refer to KIP-219. Typically, we don't include KIP numbers and JIRAs in comments for code changes. It is preferable to have comments that are self contained so that developers don't have to find the KIP to follow the code.